Libraries/LibWeb/TrustedTypes/TrustedTypePolicyFactory.cpp

```diff
@@ -134,6 +135,8 @@ Optional<Utf16String> TrustedTypePolicyFactory::get_property_type(Utf16String co
 
 TrustedTypePolicyFactory::TrustedTypePolicyFactory(JS::Realm& realm)
     : PlatformObject(realm)
+    , m_empty_html(realm.create<TrustedHTML>(realm, ""_utf16))
+    , m_empty_script(realm.create<TrustedScript>(realm, ""_utf16))
```

Comment:
suggestion: We typically allocate constant objects like this on demand when they're first requested. For example: https://github.com/LadybirdBrowser/ladybird/blob/ce03b8b3b1d9998b3936fa2f816a34270d287724/Libraries/LibWeb/HTML/Window.cpp#L1106-L1114